scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -1,5 +1,109 @@
+                {% for file, path in case.vcf_files.items() %}
+                  {% if path and file in ["vcf_snv", "vcf_cancer", "vcf_snv_research", "vcf_cancer_research"] %} <!-- support for sharing SNVs for now -->
+                    <option value="{{file}}" {{ "selected" if "research" not in file }}>{{file|replace("vcf_", "")|replace("_", " ")|upper}}</option>
+                  {% endif %}
+                {% endfor %}
+              </select>
+            </div>
+
+            <!--Gene panel selection-->
+            <div class="row d-flex justify-content-around align-items-center mt-3">
+              <label class="form-check-label" for="vcfselect">
+                Filter by panel:
+              </label>
+              <select multiple name="panels" id="panelselect" class="selectpicker" data-width="70%">>
+                {% for panel in case.panels %}
+                  <option value="{{panel.panel_id}}" {{ "selected" if panel.is_default }}>{{ panel.display_name|truncate(30, True) }} {{ panel.version }} ({{ panel.nr_genes }} genes)</option>
+                {% endfor %}
+              </select>
+            </div>
+          <!-- end of form to submit variants to beacon -->
+          <!-- case variants are already in beacon, display info -->
+          {% else %}
+            <div class="d-flex justify-content-center">
+              <table class="table table-sm">
+                <tbody>
+                  <tr>
+                    <td><strong>Created date</strong></td>
+                    <td>{{case.beacon.created_at.strftime('%Y-%m-%d %H:%M:%S')}}</td>
+                  </tr>
+                  <tr>
+                    <td><strong>Created by</strong></td>
+                    <td>{{case.beacon.user}}</td>
+                  </tr>
+                  <tr>
+                    <td><strong>VCF samples</strong></td>
+                    <td>{{ case.beacon.samples|join(", ") }}</td>
+                  </tr>
+                  <tr>
+                    <td><strong>VCF filter</strong></td>
+                    <td>{{ "gene panels" if case.beacon.panels|length > 0 else "entire VCF" }}</td>
+                  </tr>
+                  <tr>
+                    <td><strong>VCF types</strong></td>
+                    <td>{{ case.beacon.vcf_files|join(", ") }}</td>
+                  </tr>
+                </tbody>
+              </table>
+            </div>
+          {% endif %}
+      </div>
+      <div class="modal-footer d-flex justify-content-around">
+        <div>
+          {% if case.beacon %}
+            <a href="{{url_for('cases.beacon_remove', case_id=case._id)}}" class="btn btn-danger text-white">Remove all case variants from Beacon</a>
+          {% else %}
+            <button type="submit" class="btn btn-primary">Add variants to beacon</button>
+          {% endif %}
+          <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
```

Comment:
I prefere 'Cancel' over 'Close' -but that's more a matter of personal taste.

Reply:
Cancel sound better, as cancel the whole process and not just close the window, I'll change it!